src/modules/discovery/providers/discovery.service.ts

```diff
@@ -37,13 +38,13 @@ export class DiscoveryService {
           }
         }
       }
-      console.log(becknUrl.search)
-      console.log(paylaod)
+      this.logger.log("api",paylaod)
```

Comment:
which api? specify the correct log like calling search endpoint of protocol server with payload